modules/sync/sync-tax.php

```diff
@@ -76,6 +76,11 @@ protected function maybe_translate_terms( $object_id, $terms, $taxonomy, $lang )
 			$strings = array_map( 'is_string', $terms );
 			if ( in_array( true, $strings, true ) ) {
 				$terms = get_the_terms( $object_id, $taxonomy );
```

Comment:
`get_the_terms` will return an array of `WP_Terms` or `(false|WP_Error)` so it should be checked if it was actually filled with terms.